WPMedia/fusion-news-theme-blocks
# THEMES-1752: Polyfill.io removal

## Description

This PR removes the polyfill.io script in the default output type for 2 reasons:

1. It's no longer needed as all features are supported in the browsers we support. Verified against the output of `browserslist`.
2. There's a security concern with the change of ownership of the polyfill domain.

## Jira Ticket

- [THEMES-1752](https://arcpublishing.atlassian.net/browse/THEMES-1752)

## Acceptance Criteria

- Remove or replace polyfill.io script tag.

## Test Steps

1. Checkout this branch `git checkout THEMES-1752`
3. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/default-output-type-block`
4. Visit a page with all of our features on it in the browsers we support, and there should be no errors from undefined functions.
5. Support list from `browserslist`:
  - and_chr 122
  - chrome 122
  - chrome 121
  - edge 121
  - firefox 122
  - ios_saf 17.3
  - ios_saf 17.2
  - op_mob 73
  - samsung 23

## Author Checklist

        please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._



[THEMES-1752]: https://arcpublishing.atlassian.net/browse/THEMES-1752?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
